scripts/format_notebooks.py

```diff
@@ -181,6 +181,58 @@ def preprocess(self, nb, resources):
         return nb, resources
 
 
+class VersionCheckPreprocessor(preprocessors.Preprocessor):
+    metadata_tag = "VersionCheck"  # tag for added cells so that we can find them easily
+
+    try:
+        # determine the current stellargraph version by importing the library's version file
+        sys.path.append(
+            (
```

Comment:
I don't think this extra set of parens is needed? Also, if that's not enough, some part of this (maybe the `join`?) could be split out into a separate variable to make this expression a bit less complicated and explode less with `black`:

```python
library_dir = os.path.join(os.path.dirname(__file__), os.path.pardir, "stellargraph")
sys.path.append(os.path.abspath(library_dir))
```